src/gui/Src/BasicView/AbstractTableView.cpp

```diff
@@ -647,7 +647,7 @@ void AbstractTableView::leaveEvent(QEvent* event)
 void AbstractTableView::keyPressEvent(QKeyEvent* event)
 {
     int wKey = event->key();
-    if(event->modifiers())
+    if(event->modifiers() && event->modifiers() != Qt::KeypadModifier)

```

Comment:
Shouldn’t this also use `Qt::NoModifier` for consistency?

Reply:
Yes, you're right.